src/screens/SUE/SueReportScreen.tsx

```diff
@@ -305,14 +306,16 @@ export const SueReportScreen = ({
     if (
       !!sueReportData.street ||
       !!sueReportData.houseNumber ||
-      !!sueReportData.zipCode ||
+      !!sueReportData.postalCode ||
       !!sueReportData.city
     ) {
-      addressString = `${sueReportData.street}; ${sueReportData.houseNumber}; ${sueReportData.zipCode}; ${sueReportData.city}`;
+      addressString = `${sueReportData.street}; ${sueReportData.houseNumber}; ${sueReportData.postalCode}; ${sueReportData.city}`;
     }
 
     const formData = {
       addressString,
+      firstName: sueReportData?.name,
+      lastName: sueReportData?.familyName,
```

Comment:
is `...sueReportData` still needed now? this adds values to the form data we do not need, or am I wrong?

Reply:
There is still a lot of data in `sueReportData` that we need. For example `email` or `title`

Reply:
ah i see, ok